src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -49,6 +60,57 @@ internal bool AutomaticRedirection
                 _automaticRedirection = value;
             }
         }
+        internal bool UseProxy
+        {
+            get
+            {
+                if (_proxyPolicy == CurlProxyUsePolicy.UseDefaultProxy)
```

Comment:
This deviates from WinHttp which starts off with UseProxy==true and ProxyPolicy==UseWininetPolicy